aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -333,6 +333,28 @@ module std::features {
         is_enabled(BN254_STRUCTURES)
     }
 
+    /// The on-chain randomness feature.
+    ///
+    /// Lifetime: permanent
+    const ON_CHAIN_RANDOMNESS: u64 = 45;
+
+    public fun get_on_chain_randomness_feature(): u64 { ON_CHAIN_RANDOMNESS }
+
+    public fun on_chain_randomness_enabled(): bool acquires Features {
+        is_enabled(ON_CHAIN_RANDOMNESS)
+    }
+
+    /// The JWK consensus feature.
+    ///
+    /// Lifetime: permanent
+    const JWK_CONSENSUS: u64 = 46;
```

Comment:
@grao1991 i think consensus_config should cover and only cover "main consensus": things under `consensus` folder. And JWK consensus is independent, so new flag.